src/qt/window.cpp

```diff
@@ -1490,7 +1490,10 @@ bool wxWindowQt::QtHandleCloseEvent ( QWidget *handler, QCloseEvent *WXUNUSED( e
     if ( GetHandle() != handler )
         return false;
 
-    return Close();
+    if (!IsEnabled())
+        return true;
+
+    return !Close();
```

Comment:
The names here are terrible :(
FWIW the meaning of "handle" in the name of this function is opposite to the meaning in the description of `wxWindow::Close()`. Maybe this function is worth a comment, something like `// Returns true if the closing should be vetoed and false if the window should be closed`.

Reply:
I don't disagree that this section of the code is confusing, not sure where exactly the comment would be best placed. Perhaps the whole group of QtHandle... functions in qt/window.h should have a comment indicating that they return true to indicate that the event has been vetoed as they all work the same and can in theory be overridden by inheriting classes.

Reply:
> not sure where exactly the comment would be best placed

I guess at the beginning of the function / right before it?